.github/workflows/dashwallet.yml

```diff
@@ -76,8 +76,12 @@ jobs:
       
     - name: TestNet Build and Firebase Distribution
       if: github.event_name == 'push'
+      env:
+        SUPPORT_EMAIL: "${{ secrets.INTERNAL_SUPPORT_EMAIL }}"
       run: bundle exec fastlane build_distribute flavor:"_testNet3" type:"release" store_pass:"${{ secrets.SIGNING_STORE_PASS }}" version_code:"${{ env.build_number }}" comment:"Up to date Dash Wallet TestNet build" app_id:"1:1039839682638:android:3202b16d460a1a88" test_group:"qa"
       
     - name: Production Build and Firebase Distribution
       if: github.event_name == 'push'
+      env:
+        SUPPORT_EMAIL: "${{ secrets.INTERNAL_SUPPORT_EMAIL }}"
```

Comment:
`env` can be specified just once. See `run_num` in manual_distribution.yml

Reply:
Thanks, I will do this, since this method is working.